src/webhook.rs

```diff
@@ -732,11 +688,151 @@ async fn handle_comment(
 				number,
 			))))?;
 		}
+	} else if body.to_lowercase().trim() == BURNIN_REQUEST.to_lowercase().trim()
+	{
+		// Check the user is a member of the org
+		let member = github_bot
+			.org_member(&owner, &requested_by)
+			.await
+			.map_err(|e| {
+				Error::OrganizationMembership {
+					source: Box::new(e),
+				}
+				.map_issue(Some((
+					owner.to_string(),
+					repo_name.to_string(),
+					number,
+				)))
+			})?;
+		if member != 204 {
+			Err(Error::OrganizationMembership {
+				source: Box::new(Error::Message {
+					msg: format!(
+						"{} is not a member of {}; aborting.",
+						requested_by, owner
+					),
+				}),
+			}
+			.map_issue(Some((
+				owner.to_string(),
+				repo_name.to_string(),
+				number,
+			))))?;
+		}
+
+		handle_burnin_request(
+			github_bot,
+			&state.gitlab_bot,
+			&state.matrix_bot,
+			owner,
+			&requested_by,
+			&repo_name,
+			&pr,
+		)
+		.await;
 	}
 
 	Ok(())
 }
 
+async fn handle_burnin_request(
+	github_bot: &GithubBot,
+	gitlab_bot: &GitlabBot,
+	matrix_bot: &MatrixBot,
+	owner: &str,
+	requested_by: &str,
+	repo_name: &str,
+	pr: &PullRequest,
+) {
+	let make_job_link =
+		|url| format!("<a href=\"{}\">CI job for burn-in deployment</a>", url);
+
+	let unexpected_error_msg = "Starting CI job for burn-in deployment failed with an unexpected error; see logs.".to_string();
+	let mut matrix_msg = String::from("");
```

Comment:
Rust has this really fancy thing called `Option` ;) No need to treat the empty string or negative integers with special meaning.
```suggestion
	let mut matrix_msg = None;
```